src/makeCalculator.js

```diff
@@ -3,8 +3,42 @@
 /**
  * @return {object}
  */
-function makeCalculator() {
+function makeCalculator(operation, value) {
   // write code here
+
+  const calculator = {
+    operate(callback, num) {
+      callback.call(this, num);
+
+      return this;
+    },
+
+    reset: function () {
```

Comment:
[CODE KNOWLEDGE] - When creating a method in an object, you don't need to use the 'function' keyword. Use the method shorthand instead. Replace `reset: function () {` with `reset() {`.